.travis.yml

```diff
@@ -36,6 +36,12 @@ matrix:
       env: TEST_CONFIG="phpunit.xml"
     - php: 7.1
       env: REST_TEST_CONFIG="phpunit-functional-rest.xml" SYMFONY_ENV=behat SYMFONY_DEBUG=1 SF_CMD="ez:behat:create-language 'pol-PL' 'Polish (polski)'"
+    - name: "Kernel Core tests"
```

Comment:
Please say here that it refers to the behat test, from the perspective of Travis jobs list it's not obvious.